src/control/ros/Conversions.cpp

```diff
@@ -365,7 +365,11 @@ std::unique_ptr<SplineTrajectory> toSplineJointTrajectory(
     auto r1Joint = std::dynamic_pointer_cast<const R1Joint>(jointSpace);
     auto so2Joint = std::dynamic_pointer_cast<const SO2Joint>(jointSpace);
 
-    if (properties.getNumDofs() != 1 || (!r1Joint && !so2Joint))
+    if (properties.getNumDofs() == 0)
+    {
+      continue;
+    }
+    else if (properties.getNumDofs() != 1 || (!r1Joint && !so2Joint))
     {
       std::stringstream message;
       message << "Only R1Joint and SO2Joint are supported. Joint "
```

Comment:
Update comment to include WeldJoints (i.e. "Only R1Joint, SO2Joint, and WeldJoint are..."), make that the case throughout this document